### ISSUE TITLE
The loss got NAN

### ISSUE THREAD

**Issue by zaynmi** [ranjaykrishna/iq#9]
Hello Ranjay, thanks for your work.
I have run the train_iq.py using the default lr 0.001, but after 4 epochs, the loss and other metrics got Nan. Is there something wrong with the training script?
Looking forward to your reply. Thanks again.

**Comment by duchengyao** [comment]
the same issue

**Comment by ozancaglayan** [comment]
Got the same thing in the middle of epoch 2 as well.

**Comment by mahmudhasankhan** [comment]
Same, after weight decaying starts all the loss got Nan
